scripts/verify-bundle.go

```diff
@@ -14,6 +15,7 @@ import (
 
 var bundlePath = "bundle.tar.gz"
 var OrasPush = []string{"--config", "/dev/null:application/vnd.cncf.openpolicyagent.config.v1+json", fmt.Sprintf("%s:application/vnd.cncf.openpolicyagent.layer.v1.tar+gzip", bundlePath)}
+var supportedTrivyVersions = []string{"latest", "canary"} // TODO: add more versions
```

Comment:
More versions will be added later?

Reply:
I wasn't able to find a way without using Docker's API (requires a token) to be able to fetch tags. We could scrape them but I thought we can start with these first and then improve upon. 

Do you know of any way to get the tags without auth or scrape?

Reply:
We can retrieve the tags from GitHub.